client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -302,7 +306,9 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
                                 const otherReason = user ? investigation?.investigationGroupReasonByGroupId?.otherReason : '';
                                 const subStatus = investigation.investigationSubStatusByInvestigationSubStatus ?
                                     investigation.investigationSubStatusByInvestigationSubStatus.displayName :
-                                    '';
+                                    ''; 
+                                const subOccupation=  investigation?.investigatedPatientByInvestigatedPatientId?.subOccupationBySubOccupation?.displayName;
```

Comment:
space before = and delete one space after like that:
`subOccupation = investigation?.investigatedPatientByInvestigatedPatientId...`